eval/shell/job.go

```diff
@@ -313,13 +313,13 @@ func (j *Job) setupSimpleCmd(cmd *expr.ShellSimpleCmd, sio stdio) (*proc, error)
 	}
 	switch argv[0] {
 	case "cd":
-		dir := ""
+		var dir string
```

Comment:
What is generating these changes? Is it `ineffassign`? If so, sorry, I disagree with the tool. These two statements are equivalent, and an automated tool shouldn't be choosing one over the other when there's no reason for it.

Reply:
+1

Reply:
I can get rid of `ineffassign`.

Reply:
Aside from that one class of false positive (variables assigned the zero value) everything else ineffassign complains about is a valid bug. If ineffassign isn't run via Travis, it should be run by hand frequently.